### DISCUSSION TITLE
Using Multiple Editors

### DISCUSSION THREAD

**Discussion by Jxmedia** [tinymce/tinymce-react#487] [Q&A]
Hello everyone,

So I'm using "@tinymce/tinymce-react": "^4.3.0",

And for the life of me I cannot figure out how to use multiple editors tied to different selectors. In the tinyMce docs it says I can use a selector ID for targeting specific editors but for the react package I can't figure out how to change this:

editorRef.current.getContent()

to

tinymce.get("selectorHere").getContent();

---------------------

My problem is that I have 12 editors on the page but when saving the content it only refers to the last one or "current", any help would be appreciated thanks

**Comment by lahipa** [comment]
Assuming you has created multiple input value change handle. You can create a component that will handle input processing with TinyMce, then load the component you just created in a page.
